pandas/tests/extension/json/array.py

```diff
@@ -27,7 +27,7 @@
 class JSONDtype(ExtensionDtype):
     type = abc.Mapping
     name = "json"
-    na_value = UserDict()
```

Comment:
@simonjayhawkins i know ive asked you this before, but why cant mypy figure this out on its own?

Reply:
the revealed type for na_value is `Revealed type is 'collections.UserDict[Any, Any]'`.

mypy asks for more info to narrow the `Any`